x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_validators.test.tsx

```diff
@@ -34,6 +34,12 @@ describe('Output form validation', () => {
       expect(res).toBeUndefined();
     });
 
+    it('should work with hostnames using uppercase letters', () => {
+      const res = validateKafkaHosts(['tEsT.fr:9200', 'TEST2.fr:9200', 'teSt.fR:9999']);
```

Comment:
did you mean to test `validateLogstashHosts`, since the change is there?

Reply:
Yes, yes I did thank you :D